.github/stale.yml

```diff
@@ -0,0 +1,42 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 30
+
+# Issues with these labels will never be considered stale
+exemptLabels:
```

Comment:
should we add `hold` & `WIP` in exemptLabels?

Reply:
Not sure... Looks like PRs in K8s community will be marked stale and closed even have `hold` or `WIP` labels.